src/pineko/kfactor.py

```diff
@@ -9,15 +9,11 @@
 from pineappl import import_only_subgrid
 
 from . import scale_variations
+from .scale_variations import orders_as_tuple
 
 DEFAULT_PDF_SET = "NNPDF40_nnlo_as_01180"
 
 
-def factgrid(subgrid):
-    """Return the array of the factorization scales squared from a subgrid."""
-    return np.array([mu2.fac for mu2 in subgrid.mu2_grid()])
-
-
 def rengrid(subgrid):
     """Return the array of the renormalization scales squared from a subgrid."""
     return np.array([mu2.ren for mu2 in subgrid.mu2_grid()])
```

Comment:
I would inline this - there is a single use and the function is almost trivial

Reply:
Sure this is possible.